.gitignore

```diff
@@ -18,3 +18,4 @@
 /tempodb/encoding/benchmark_block
 private-key.key
 integration/e2e/e2e_integration_test[0-9]*
+.tempo.yaml
```

Comment:
what's this?

Reply:
It is my local config to debug Tempo. I can leave it out

Reply:
it's fine. just interested